mux/bindings/java/scripts/build.sh

```
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+cd "$ROOT"
+rm -rf out
+mkdir -p out
+javac --release 17 -d out $(find src tests -name '*.java' | sort)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)